### DISCUSSION TITLE
The position of "Switch branches/tags" panel is not fixed.

### DISCUSSION THREAD

**Discussion by ZhanZiyuan** [community/community#121305] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello. I don't know whether it appears to be a bug or not: the "Switch branches/tags" panel scrolls with the browser's scrollbar.
Screenshots from my private repository. Browsing with Microsoft Edge version `124.0.2478.67` (64 bit).

![01](https://github.com/community/community/assets/154674404/aa5cc42a-c5e0-48d4-9086-984c329a98fe)

![02](https://github.com/community/community/assets/154674404/a4bc2724-d353-4dff-8f7d-ce6741888620)

**Comment by cyrolux123** [comment]
Did you googled?

**Reply by ZhanZiyuan** [reply]
Hello. Thanks for your reminding.
Sorry, I didn't find many web pages related to "GitHub Switch branches moving" or "GitHub Switch branches/tags scrolls".
